integration-tests/drivers/TestDriver063/.gitignore

```diff
@@ -59,5 +59,5 @@ buck-out/
 /ios/Pods/
 
 # From test script
-src/
-tests/
+/src/
```

Comment:
This was wiping out /android/app/src/ 😦 What'e even the point of `*/foo` in .gitignore if `foo` does the same thing?